CONTRIBUTING.md

```diff
@@ -1,46 +1,109 @@
 # Development
 
-Below lists a few commands for building and running a local copy of VolView.
+## Prerequisites
 
-To prepare your environment after cloning the repo, run:
+Before you begin, make sure your environment matches the following versions:
+
+- **Node.js**: >= 18.20.0 (20.x LTS recommended)  
+- **npm**: >= 9.x (npm 10+ works with Node 20)
+
+Check your versions:
 
+```bash
+node --version
+npm --version
 ```
+
+> **Tip:** Use `nvm` or `volta` to manage and switch Node.js versions.
+
+For reproducible installs, use:
+
+```bash
+npm ci
+```
+
+If you encounter errors about `package-lock.json` schema, regenerate it:
+
+```bash
+rm package-lock.json
 npm install
```

Comment:
Deleting package-lock.json will cause problems to get that regenerated package-lock.json merged back into main.